x-pack/plugins/security/server/usage_collector/security_usage_collector.test.ts

```diff
@@ -8,8 +8,8 @@ import { createConfig, ConfigSchema } from '../config';
 import { loggingSystemMock } from 'src/core/server/mocks';
 import { TypeOf } from '@kbn/config-schema';
 import { usageCollectionPluginMock } from 'src/plugins/usage_collection/server/mocks';
+import { createCollectorFetchContextMock } from 'src/plugins/usage_collection/server/usage_collection.mock';
```

Comment:
__question:__ why can't we export\import `createCollectorFetchContextMock` from `src/plugins/usage_collection/server/mocks` like we do for `usageCollectionPluginMock`? I believe we have a convention regarding exporting/importing mocks only from `server/mocks` and `public/mocks`.

Reply:
Great question! [Fixed](https://github.com/elastic/kibana/pull/79595/files#diff-216dfdbca9c4d11af293cb229c21ea98e1c146c8de1779700e4fcaaeb45ee94cR10-R13)